accord-core/src/main/java/accord/utils/SimpleBitSet.java

```diff
@@ -419,6 +419,19 @@ private <P1, P2, P3, P4> void reverseForEach(int index, long mask, P1 p1, P2 p2,
         }
     }
 
+    public void intersect(SimpleBitSet with)
```

Comment:
this method doesn't do anything... as long as `this.size == with.size` we spend time computing things, but we don't actually do anything with what we compute.

IntelliJ also says this is dead code, can we remove?

Reply:
It does do something (it updates one of the two arrays) but yes it’s dead code so will delete - might even be incomplete anyway as from an abandoned change I otherwise deleted.